### PR TITLE
Register samyakgptui.is-an.app

### PR DESCRIPTION
**Pull request by Sammy970**
A website with a Clean UI that would work with help you interact with various different GPT API Models

 `Verification Records`

Type | Name | Value
-- | -- | --
TXT | _vercel | vc-domain-verify=samyakgptui.is-an.app,da6ca182d4f1d64f6b3b

### Link to Website

Link: https://test-deploy-weld.vercel.app/

---

- [ ] Toggle this checkbox to re-run the stargazer checking

### CODE REVIEW

**Comment by Sammy970** [comment]
I did everything same as, Rygent did.
My app is hosted on vercel.com

Rygent PR - https://github.com/tarampampam/free-domains/pull/300

How to solve this DNSControl check ?

File: domains/samyakgptui.is-an.app.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  description: 'A ChatBOT with clean UI for interacting with various different GPT models at different EndPoints.', // describe your project in this field
+  domain: 'is-an.app', // aka "root-domain". select between '1bt.uk' and 'is-an.app'
+  subdomain: 'samyakgptui', // desired subdomain name
+  owner: {
+    repo: 'https://github.com/Sammy970/SamyakGPTUI',
+    email: 'jainsamyak2002.sj@gmail.com',
+  },
+  record: {
+      CNAME: 'cname.vercel-dns.com'
+  }
```
Comment (tarampampam) [inline domains/samyakgptui.is-an.app.js:11]:
```suggestion
  }
})
```

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Comment by Sammy970** [comment]
I did the changes, but still the check if failed

**Comment by Sammy970** [comment]
The checks are cleared now. 😁

**Review by Sammy970 — COMMENTED** [review]
Changes done

**Pull request merged**